common/shell-local/config.go

```diff
@@ -56,6 +56,11 @@ type Config struct {
 }
 
 func Decode(config *Config, raws ...interface{}) error {
+	//Create passthrough for winrm password so we can fill it in once we know it
+	config.Ctx.Data = &EnvVarsTemplate{
```

Comment:
I wanted to be able to interpolate any other env vars when the template was being read, in case someone is setting an env var for the shell that they're reading from template variables.